test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -0,0 +1,96 @@
+Describe "Start-Transcript, Stop-Transcript tests" -tags "CI" {
+    
+    BeforeAll {
+
+        function ValidateTranscription {
+            param
+            (
+                [string] $scriptToExecute,
+                [string] $outputFilePath,
+                [switch] $Append,
+                [switch] $NoClobber,
+                [string] $expectedError
+            )
+            
+            if($Append -or $NoClobber)
+            {
+                #Add sample text to the file
+                $content = "This is sample test!"
+                $content | Out-File -FilePath $outputFilePath
+                Test-Path $outputFilePath | Should be $true
+            }
+            #execute script
+            $ps = [powershell]::Create()
+            $ps.addscript($scriptToExecute).Invoke()
+            $ps.commands.clear()
+
+            if($expectedError)
+            {
+                $ps.hadErrors | Should be $true
+                $ps.Streams.Error.FullyQualifiedErrorId | Should be $expectedError
+            }
+            else 
+            {
+                $ps.addscript("Get-Date").Invoke()
+                $ps.commands.clear()
+                $ps.addscript("Stop-Transcript").Invoke()
+
+                Test-Path $outputFilePath | Should be $true
+                $outputFilePath | should contain "Get-Date"
+                if($Append)
+                {
+                    $outputFilePath | Should contain $content
+                }
+            }
+        }
+        ## function ends here
+
+        $transcriptFilePath = join-path $TestDrive "transcriptdata.txt"         
+    }
+    
+
+    BeforeEach {
+        Remove-Item $transcriptFilePath -ErrorAction SilentlyContinue
+    }
```

Comment:
It seems the test transcript file will not be cleaned up after the last test case.